doc/cdrwtool.1

```diff
@@ -135,11 +135,29 @@ variable and fixed packet sizes respectively.
 .IP "\fB\-o \fIoffset\fP"
 Set write offset.
 
+.SH BUGS
+Many modern drives refuse on the preparations to format new,
+blanked, or appendable CD-RW media. This causes a message like
+.nf
+  "Command failed: 55 ... - sense ..."
+.fi
+.br
+The remedy is to use a CD-capable burn program for writing a session
+and closing the medium. For example by using any of "cdrecord", 
+"wodim", "cdrskin", or "xorriso -as cdrecord" as content of
+variable "prog" in:
+.nf
+  prog="xorriso -as cdrecord"
+  drive="/dev/sr0"
+  dd if=/dev/zero bs=1M count=10 | $prog -v -eject dev="$drive" -
+.fi
```

Comment:
Put code into manpage example block, it can be filled by `.EX` and `.EE` manpage macros. Indentantion then by `.in` macro:
```suggestion
.PP
.in +4n
.EX
prog="xorriso -as cdrecord"
drive="/dev/sr0"
dd if=/dev/zero bs=1M count=10 | $prog -v -eject dev="$drive" -
.EE
.in
```

I hope that I still remember troff language and manpage macros :-)